platform/platform-filter-response/src/main/java/org/codice/ddf/platform/response/filter/ResponseFilter.java

```diff
@@ -59,16 +60,17 @@
             X_CONTENT_SECURITY_POLICY + "=" + DEFAULT_CONTENT_SECURITY_POLICY,
             CACHE_CONTROL + "=" + DEFAULT_CACHE_CONTROL_VALUE);
 
-    @Override
-    public void init(FilterConfig filterConfig) throws ServletException {
-        LOGGER.debug("Initializing Response Security Filter.");
+    // Index paths (such as /search/catalog/ or /admin/) should never be cached
+    private void disableCachingForHTML(HttpServletRequest request, HttpServletResponse response) {
```

Comment:
Sounds good, can you work with me on this update?  Trying to get it in for the 2.10.2 release.